webserver/templates/user/profile.html

```diff
@@ -11,6 +11,13 @@ <h2 class="page-title">{{ user.musicbrainz_id }}</h2>
       <b><a href="https://musicbrainz.org/user/{{ user.musicbrainz_id }}">See profile on MusicBrainz</a></b>
     </div>
 
+    {% if have_listen_count %}
+      <h3> Listen Count: {{ listen_count }}  </h3>
+      <p> Note: This is an approximate value.<a href = "{{ url_for("user.profile", user_name=user.musicbrainz_id, exact = 'y') }}"> Get exact listen count! </a> </p>
```

Comment:
It seems strange to have spaces inside the tags. I'd remove them. It's especially odd with the `<a>` link - here the link will extend out into the spaces either side of the phrase, which is not normal.